### ISSUE TITLE
Blazored Modal doesn't appear in IE

### ISSUE THREAD

**Issue by DSBhogal** [Blazored/Modal#181]
Building an ECommerce platform using server side blazor. Currently utilising Blazored modal which works great on Chrome but doesn't show on IE? 

Hits this method fine, no issues when hitting 'ModalService.Show' but just doesn't do anything:

![image](https://user-images.githubusercontent.com/33750091/85268911-0e269300-b46f-11ea-98ea-923053b3c864.png)

**Comment by larsk2009** [comment]
Could you provide a minimal reproduction of the issue and also which version of IE you are using?

**Comment by DSBhogal** [comment]
The version of IE that I'm using is IE 11 specifically 11.900.18362.0. I can provide you with some screenshots of when the method is hit?, unsure how else to provide you with a minimal reproduction of the issue

**Comment by larsk2009** [comment]
Do you also have the issue when using the Sample in this repository?

**Comment by DSBhogal** [comment]
Just tried with the sample repository and that has the same issue, it doesn't work on IE

**Comment by larsk2009** [comment]
I had a look, and it also didn't work for me. I am pretty sure it is a IE11 thing. If I look at the [supported browsers](https://docs.microsoft.com/en-us/aspnet/core/blazor/supported-platforms?view=aspnetcore-3.1) page for Blazor, IE11 is supported, but you need to add polyfills. If I add [this polyfill](https://github.com/Daddoon/Blazor.Polyfill) the modal does show up but the CSS is not correct. This could be fixed.

So in summary: to get the modal to work in IE11:

- End user needs to include [the polyfill](https://github.com/Daddoon/Blazor.Polyfill)
- The `blazored-modal-overlay` class needs a small css change:

``` css
.blazored-modal-overlay {
/*Original CSS goes here*/
    top: 0;
    left: 0;
}
```
- Probably the readme needs to be updated to inform users about this.

@chrissainty is this something you want to add? IE11 is only supported by blazor Server and not by webassembly so it would still not always work on IE11.

**Comment by chrissainty** [comment]
@larsk2009 yes it would be good to have the modal work in all places Blazor does. The changes you’ve identified are quite small and non-breaking. As long as they don’t break anything on wasm, let’s get a PR together and get them merged.

**Comment by larsk2009** [comment]
I made a PR to fix this issue. @chrissainty I just wanted to say, I hope you don't mind that I am responding to many issues on here. I really like this component and want to help you with it. Just don't want to overstep you if you understand me.

**Comment by DSBhogal** [comment]
Much appreciated for the help, will we be expecting an update to the NuGet package or is worth me implementing the fixes myself?

**Comment by chrissainty** [comment]
@DSBhogal We probably won't have an update out for a few days. The next release will likely be a breaking change so I want to pack in as many features as possible. So it might be worth adding the workarounds yourself for the short term.

**Comment by chrissainty** [comment]
@larsk2009 No it's fine, I'm happy for the help and you're doing a great job! 😀

**Comment by DSBhogal** [comment]
> @DSBhogal We probably won't have an update out for a few days. The next release will likely be a breaking change so I want to pack in as many features as possible. So it might be worth adding the workarounds yourself for the short term.

No worries, would you like me to close the issue?

**Comment by chrissainty** [comment]
@DSBhogal No it's fine, the issue will be closed automatically when I merge the PR

**Comment by chrissainty** [comment]
Resolved by #182
